docs/section10/10-1.md

```diff
@@ -0,0 +1,41 @@
+# Airside systems – Rule 10-1  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False    
+**Rule ID:** 10-1  
+ 
+**Rule Description:** The proposed design includes humidification (which does not comply with 90.1-2019 Section 6.5.2.4) and the baseline building design has been modeled with adiabatic humidification, as expected.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_zones_hvac_system_humid_p()
```

Comment:
Not sure we need this function - see below.

Reply:
@claperle and @mkarpman i believe we would use #365 to get list of all HVAC systems and then in this RDS identify the HVAC systems in that list where humidification type is adiabatic or other (if null or NONE then no humidification): `if humidification_type_p in ["ADIABATIC", "OTHER"]:`

If yes, then that makes sense to me.

dont think we need #258 then, correct?